data/docs/alerts-management/terraform-provider-signoz.mdx

```diff
@@ -0,0 +1,240 @@
+            "dataSource" : "metrics",
+            "disabled" : false,
+            "expression" : "A",
+            "filters" : {
+              "items" : [],
+              "op" : "AND"
+            },
+            "groupBy" : [
+              {
+                "dataType" : "string",
+                "isColumn" : false,
+                "isJSON" : false,
+                "key" : "k8s_node_name",
+                "type" : "tag"
+              }
+            ],
+            "limit" : 0,
+            "offset" : 0,
+            "pageSize" : 0,
+            "queryName" : "A",
+            "reduceTo" : "avg",
+            "spaceAggregation" : "avg",
+            "stepInterval" : 60,
+            "timeAggregation" : "avg"
+          }
+        },
+        "chQueries" : {
+          "A" : {
+            "disabled" : false,
+            "query" : ""
+          }
+        },
+        "panelType" : "graph",
+        "promQueries" : {
+          "A" : {
+            "disabled" : false,
+            "query" : ""
+          }
+        },
+        "queryType" : "builder",
+        "unit" : "bytes"
+      },
+      "matchType" : "1",
+      "op" : "1",
+      "selectedQueryName" : "A",
+      "target" : 10,
+      "targetUnit" : "gbytes"
+    }
+  )
+  severity           = "info"
+  broadcast_to_all   = false
+  preferred_channels = []
+  rule_type          = "threshold_rule"
+  description        = "An example alert created by Terraform"
+  eval_window        = "5m0s"
+  frequency          = "1m0s"
+  labels = {
+    "environment" = "test"
+  }
+}
+   ```
+This configuration defines:
+- condition: Trigger alerts if `k8s_node_memory_rss` is missing for 10 seconds.
+- metric: Aggregates the average value of the `k8s_node_memory_rss` metric grouped by Kubernetes node name.
+- interval: Evaluates the metric every 1 minutes with 5-minutes evaluation window.
+
+### Step 5: Check Correctness of Definition
+
+After defining your alert, validate its syntax by running `validate` command:
+```bash
+terraform validate
+```
+If the resource is defined correctly, you will see the message: `Success! The configuration is valid.`
+
+In case you get any error, fix your definition by referencing the console output:
+
+![Validate error](../../../public/img/blog/common/terraform_provider_validate_error.webp)
+
+The error indicates that:
+- the required argument alert_type` was not found.
+- an unexpected argument `alert_typ was passed.
+
+Fix the typo that caused this error:
+```diff
+ resource "signoz_alert" "example" {
+   alert      = "Example alert"
+-  alert_typ = "METRIC_BASED_ALERT"
++  alert_type = "METRIC_BASED_ALERT"
+   condition = jsonencode(
+     {
+       "absentFor" : 10,
+```
+
+### Step 6: Provision SigNoz Alert to SigNoz Cloud
+
+When you're ready, `plan` your Terraform module to see diffs between your code and resources on the remote (SigNoz Cloud):
+
+![Plan](../../../public/img/blog/common/alerts_terraform_provider_plan.webp)
```

Comment:
Please put all the images in this folder rather than the blog folder - https://github.com/SigNoz/signoz-web/tree/main/public/img/docs/alerts

Reply:
fixed: 01280eb5